### PR TITLE
couple fixes of Quantum-net-creation procedure

### PR DESCRIPTION
**Pull request by xenolog**

### CODE REVIEW

**Comment by aglarendil** [comment]
+1

File: deployment/puppet/osnailyfacter/manifests/cluster_simple.pp [hunk 1]
```diff
@@ -198,6 +198,7 @@
         cinder_rate_limits      => $cinder_rate_limits,
         horizon_use_ssl         => $horizon_use_ssl,
         nameservers             => $::dns_nameservers,
+        primary_controller      => true,
```
Comment (mihgen) [inline deployment/puppet/osnailyfacter/manifests/cluster_simple.pp:201]: Don't we need similar thing in cluster_ha.pp ?

Comment (xenolog) [inline deployment/puppet/osnailyfacter/manifests/cluster_simple.pp:201]: we need this flag for simplify OS::controller manifest code

**Comment by xenolog** [comment]
tested by Andrey Danin as centos/quantum/simple/gre

waiting tests on nova-network configuration

**Comment by mihgen** [comment]
tested on nova-network with VLANManager.

**Pull request merged**